plugin/dns64/dns64.go

```diff
@@ -0,0 +1,188 @@
+// Package dns64 implements a plugin that performs DNS64.
+package dns64
+
+import (
+	"context"
+	"errors"
+	"net"
+	"time"
+
+	"github.com/coredns/coredns/plugin"
+	"github.com/coredns/coredns/plugin/pkg/nonwriter"
+	"github.com/coredns/coredns/plugin/pkg/response"
+	"github.com/coredns/coredns/plugin/pkg/upstream"
+	"github.com/coredns/coredns/request"
+	"github.com/coredns/proxy"
```

Comment:
proxy need to be removed; also wonder why it's in here in the first place? It queries by itself?

Reply:
Done, removed proxy from the query path, now uses the normal upstream to fetch the data for translation.